### PR TITLE
Feature/aolivier web display

### PR DESCRIPTION
**Pull request by aolivier23**
"Beta" version of web-based event display.  It draws LineSegments, MC trajectories, and the geometry for now.  It works on Chrome on MacOS and Linux as well as Firefox on Linux.  Each user starts their own ART job and points their browser at the magic port number.

This commit is just enough to get us started.  Future features will include supporting Safari, supporting Track(Segment)s and the ARICH, and a more robust application design.  So please leave this branch open after the pull request!

### CODE REVIEW

File: EventDisplay/WebDisplay/justRunSSDReco.fcl
Comment (gavinsdavies) [inline EventDisplay/WebDisplay/justRunSSDReco.fcl:1]: i would suggest this follow some convention for naming like just_run_ssd_reco_job.fcl  (ending with job.fcl distinguishes it from config files.)

**Review by gavinsdavies — APPROVED** [review]
i'm just excited to try this out. 
I'd recommend turning the README into a wiki page as well as logging some of those TODOs as Issues (adding an evd tag to keep them together).
i assume folks using the port is fine -- wonder what a peak for bandwidth is, though at least for now, many users at once probably not a big concern.

File: EventDisplay/WebDisplay/justRunSSDReco.fcl
Comment (aolivier23) [reply EventDisplay/WebDisplay/justRunSSDReco.fcl:1]: Oops, I didn't mean for that reconstruction .fcl to get into the commit.  I'll remove it soon.  We have better .fcls for this.

I'll convert the README, thanks.

The peak bandwidth is a good point.  Data only gets transferred when you ask for a new event.  The rendering happens all in the browser which should help.  With the planned upgrade, we will gain the ability to swap out the back end for something like Apache with an ART-based CGI "script" which might help if we develop serious bandwidth problems.

**Review by jonpaley — APPROVED** [review]
I tried this yesterday, it works great!

**Pull request merged**